_client/src/Ui/Views/UsersView.cpp

```diff
@@ -69,13 +79,17 @@ void UsersView::fillTable(const QVector<DisplayData::User> &users)
     m_dataTable->setRowCount(users.size());
     m_dataTable->setColumnCount(DisplayData::User::VAR_COUNT);
     QStringList headers = {"ID", "Username", "Password", "Role ID", "Role Name"};
+    m_dataTable->setColumnWidth(2, 550);
```

Comment:
Don't use "magic numbers"
Create global const variable in special headers

You also can consider creating enum or something to make it easier to read column numbers